zebedee-cms/src/main/java/com/github/onsdigital/zebedee/keyring/KeyringCache.java

```diff
@@ -11,7 +8,7 @@
  * {@link com.github.onsdigital.zebedee.model.Collection}s. Provides methods to add, retrieve and remove keys to/from
  * the keyring.
  */
-public interface KeyringCache {
+public interface KeyringCache extends SchedulerKeyCache {
```

Comment:
The name `KeyringCache` seems a bit more generic than `SchedulerKeyCache`. The `SchedulerKeyCache` doesn't seem to have anything specific to scheduling. Consider if the naming is right for these interfaces.